e2e-tests/nuts-network/direct-wan/run-test.sh

```diff
@@ -17,7 +17,8 @@ echo "------------------------------------"
 echo "Performing assertions..."
 echo "------------------------------------"
 # Wait for Nuts Network nodes to build connections
-sleep 5
+waitForDiagnostic "nodeA" connected_peers_count 1
+waitForDiagnostic "nodeB" connected_peers_count 1
 # Assert that node A is connected to B and vice versa using diagnostics. It should look something like this:
 assertDiagnostic "http://localhost:11323" "connected_peers_count: 1"
 assertDiagnostic "http://localhost:21323" "connected_peers_count: 1"
```

Comment:
does it still need the assertions too? (same in other tests)

Reply:
No, don't know how I missed those